src/lib.rs

```diff
@@ -181,7 +181,7 @@ pub fn debugger_test(attr: TokenStream, item: TokenStream) -> TokenStream {
             // On Windows, use the IsDebuggerPresent API to check if a debugger is present
             // for the current process. https://docs.microsoft.com/en-us/windows/win32/api/debugapi/nf-debugapi-isdebuggerpresent
             #[cfg(windows)]
-            extern "stdcall" {
+            extern "system" {
                 fn IsDebuggerPresent() -> i32;
```

Comment:
You could use [windows-link](https://crates.io/crates/windows-link) and avoid the implicit dependency on the Windows SDK here.